pilot/pkg/model/authorization.go

```diff
@@ -65,8 +65,17 @@ type AuthorizationPoliciesResult struct {
 	Audit  []AuthorizationPolicy
 }
 
+type ProxyInfo struct {
```

Comment:
@keithmattix  is this what you had in mind when you said to pass the proxy info as a struct?

Reply:
Yeah something like that! Just a data structure to hold the matching criteria

Reply:
Yeah I think you can probably use `workloadSelectionOpts` instaed

Reply:
I will keep it as we may need additional workload selection options in the future.